pkg/storaged/stratis-details.jsx

```diff
@@ -364,17 +364,17 @@ export const StratisPoolDetails = ({ client, pool }) => {
 
     function create_fs() {
         dialog_open({
-            Title: _("Create Filesystem"),
+            Title: _("Create filesystem"),
```

Comment:
If we break translatable strings, then let's please fix them properly, and move to "file system".

Reply:
we use "file system" once in the whole Cockpit and "filesystem" dozens of times. Are you sure I should change this one occurrence to "file system"?

Reply:
IMHO *if* we touch a translatable string, we should better just fix it at the same time.

Reply:
Done